src/components/Strategies/StrategyList.tsx

```diff
@@ -5,13 +5,25 @@ import { PlusOutlined } from '@ant-design/icons';
 import styled from 'styled-components';
 
 import { IStrategy } from './../../types';
+import { strategyPathKey } from '../../constants';
+import { useAuth } from '../../hooks/index';
```

Comment:
What I meant by `index` is just use `/hooks`. Look at all the other imports and even the import you made in `Auth` in this same PR